### ISSUE TITLE
How do I get rid of MacBook's name? and weird background color?

### ISSUE THREAD

**Issue by tr1s** [tylerreckart/hyperzsh#21]
I'm setting up my new laptop and hyperzsh no longer works as expected. I get the full name of my macbook before the ~ which is annoying and takes up space. Do you know how I get rid of it?

Also I'm getting these weird background colours when I'm tabbing through options. I can't tell what this is coming from, it happens on Hyper.js and VSCode terminal. I've kept all settings the same (as far as I know). Has anyone ran into this before?

Old Laptop Proper Way
<img width="405" alt="Screenshot 2019-03-13 14 54 35" src="https://user-images.githubusercontent.com/19172250/54306560-fdb31780-459f-11e9-9fda-cb2ab5a35d27.png">

New Laptop Bad Way
<img width="421" alt="Screenshot 2019-03-13 14 49 28" src="https://user-images.githubusercontent.com/19172250/54306287-664dc480-459f-11e9-840b-471b69c78ae2.png">

**Comment by tylerreckart** [comment]
The hostname being displayed at the top of the window is part of the theme's intended functionality. As far as the colors go, I would refer to your terminal's color settings to get that sorted out.

**Comment by tr1s** [comment]
@tylerreckart I'm using hyperzsh on my old laptop though. It's the exact same theme with the exact same settings for both .zshrc and hyper.js. If I go look at my .zshrc on my old machine it says "hyperzsh" for the theme. So how come it's different now, do I just have a later version with new defaults somehow? **Is there no way to get rid of it apart from switching themes?**

I was thinking that about the terminal colour settings, but then why would both hyper and vscode terminal happen to apply the same background style? The only way that makes sense is if hyperzsh in the .zshrc is affecting both hyper and vscode terminal (as it should), and applying these styles. Hence why I'm asking here.

**Comment by tylerreckart** [comment]
Can you list the commit SHA for when you cloned the repo on your machine with the correct coloring and the SHA for the repo on your new machine? That will help me track down exactly where this change may have been made.

**Comment by tylerreckart** [comment]
Closing due to lack of response from issue author.

**Comment by tr1s** [comment]
@tylerreckart I don't know how to do that or what you're talking about, so I just let it go. I'm using a [different theme](https://github.com/robbyrussell/oh-my-zsh/wiki/Themes) "zhann" that only has the one squiggle.

I didn't clone any repo, I just followed the instructions and ran `wget -O $ZSH_CUSTOM/themes/hyperzsh.zsh-theme https://raw.githubusercontent.com/tylerreckart/hyperzsh/master/hyperzsh.zsh-theme`. Who knows how I did it 2 years ago on my old machine, I assume the same way.

If you feel like helping me with detailed instructions how to figure out what went wrong I'd be glad to help, but otherwise I'll just stick with this new theme for now. I can definitely assure you I haven't done anything to either `hyper.js` or `.zshrc` besides changing the theme on the `.zshrc`. So I was just confused as to why it's behaving so different now. Maybe it is just me if no one else has mentioned this issue though... 🤷🏽‍♂️

Thanks

**Comment by kscarlett** [comment]
FWIW, I had the same issue just now and I fixed it by copy-pasting the theme from my old install. I didn't clone the theme originally (followed the instructions with wget too), but the file was downloaded on 11 May 2018, so the change must have happened after that point.

If I had to hazard a guess, I'd say #16 is most likely to have caused it, as it is related to the username and hostname.

Edit for clarity: I mean the hostname, not the colours (but those might also be affected, I just didn't check).
